packages/terra-slide-panel-manager/tests/wdio/slide-panel-manager-spec.js

```diff
@@ -0,0 +1,167 @@
+    Terra.should.beAccessible({ viewports });
+  });
+
+  describe('Disclose Small', () => {
+    before(() => browser.url('/#/raw/tests/slide-panel-manager/slide-panel-manager-default')
+        .click('#root-component .disclose-small'));
+
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.beAccessible({ viewports });
+  });
+
+  describe('Disclose Medium', () => {
+    before(() => browser.url('/#/raw/tests/slide-panel-manager/slide-panel-manager-default')
+        .click('#root-component .disclose-medium'));
+
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.beAccessible({ viewports });
+  });
+
+  describe('Disclose Large', () => {
+    before(() => browser.url('/#/raw/tests/slide-panel-manager/slide-panel-manager-default')
+        .click('#root-component .disclose-large'));
+
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.beAccessible({ viewports });
+  });
+
+  describe('Disclose Huge', () => {
+    before(() => browser.url('/#/raw/tests/slide-panel-manager/slide-panel-manager-default')
+        .click('#root-component .disclose-huge'));
+
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.beAccessible({ viewports });
+  });
+
+  describe('Disclose Fullscreen', () => {
+    before(() => browser.url('/#/raw/tests/slide-panel-manager/slide-panel-manager-default')
+        .click('#root-component .disclose-fullscreen'));
+
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.beAccessible({ viewports });
+  });
+
+  describe('Nested Disclose', () => {
+    before(() => {
+      browser.setViewportSize(Terra.viewports('large')[0]);
+      browser.url('/#/raw/tests/slide-panel-manager/slide-panel-manager-default');
+
+      browser.click('#root-component .disclose-large');
+
+      browser.waitForVisible('[class*="slide-group"] #DemoContainer-1 .disclose-tiny', 500);
+      browser.click('[class*="slide-group"] #DemoContainer-1 .disclose-tiny');
+    });
+
+    Terra.should.matchScreenshot();
+    Terra.should.beAccessible();
+  });
+
+  describe('Disclosure - Dismiss', () => {
+    before(() => {
+      browser.setViewportSize(Terra.viewports('large')[0]);
+      browser.url('/#/raw/tests/slide-panel-manager/slide-panel-manager-default');
+
+      browser.click('#root-component .disclose-large');
+
+      browser.waitForVisible('[class*="slide-group"] #DemoContainer-1 .dismiss', 500);
+      browser.click('[class*="slide-group"] #DemoContainer-1 .dismiss');
+    });
+
+    Terra.should.matchScreenshot();
+    Terra.should.beAccessible();
+  });
+
+  describe('Disclosure - Close Disclosure', () => {
+    before(() => {
+      browser.setViewportSize(Terra.viewports('large')[0]);
+      browser.url('/#/raw/tests/slide-panel-manager/slide-panel-manager-default');
+
+      browser.click('#root-component .disclose-large');
+
+      browser.waitForVisible('[class*="slide-group"] #DemoContainer-1 .close-disclosure', 500);
```

Comment:
Would it be valuable to capture a screenshot before and after the discloser event?